CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,343 @@
+
+## Acknowledgment and Responding to Immediate Needs
+
+CoC Committee members and/or event staff will attempt to ensure your safety and help with any immediate needs. The CoC Committee will make every effort to **acknowledge receipt within 24 hours** (and we'll aim for much more quickly than that).
+
+## Reviewing the Report
+
+The CoC Committee will make all efforts to **review the incident within three days** and determine:
+
+- Whether this is an ongoing situation, or if there is a threat to anyone's physical safety
+- What happened
+- Whether this event constitutes a code of conduct violation
+- Who the bad actor was, if any
+
+## Contacting the Person Reported
+
+After the CoC Committee has had time to review and discuss the report, someone will attempt to contact the person who is the subject of the report to inform them of what has been reported about them. We will then ask that person for their account of what happened.
+
+## Response and Potential Consequences
+
+Once the CoC Committee has completed our investigation of the report, we will make a decision as to how to respond. The person making a report will not normally be consulted as to the proposed resolution of the issue, except insofar as we need to understand how to help them feel safe.
+
+Potential consequences for violating the conda-forge Organization code of conduct include:
+
+- Nothing (if we determine that no violation occurred)
+- Private feedback or reprimand from the CoC Committee to the individual(s) involved
+- Warning the person to cease their behavior and that any further reports will result in sanctions
+- A public announcement that an incident occurred
+- Mediation (only if both reporter and reportee agree)
+- An imposed vacation (e.g. asking someone to "take a week off" from a mailing list)
+- A permanent or temporary ban from some or all the conda-forge Organization spaces (mailing lists, GitHub repositories / feedstocks, in-person events, etc.)
+- Assistance to the complainant with a report to other bodies, for example, institutional offices or appropriate law enforcement agencies
+- Removing a person from the conda-forge Organization membership or other formal affiliation
+- Publishing an account of the harassment and calling for the resignation of the alleged harasser from their responsibilities (usually pursued by people without formal authority: may be called for if the person is the event leader, or refuses to stand aside from the conflict of interest, or similar)
+- Any other response that the CoC Committee deems necessary and appropriate to the situation
+
+No one espousing views or values contrary to the standards of our code of conduct will be permitted to hold any position representing the conda-forge Organization, including volunteer positions. The CoC Committee has the right and responsibility to remove, edit, or reject comments, commits, code, website edits, issues, and other contributions that are not aligned with this code of conduct.
```

Comment:
Not sure if people consider this worth specifying, but removing merged commits is highly disruptive, especially as this affects the traceability of packages have been built on these. I'm not saying we should rule this out, but following the CoC to the letter here says "Committee has the responsibility to remove commits that are not aligned with CoC".

I think such removals should have to meet a higher bar than "in scope under the strictest interpretation of the CoC". For example, if someone finds a swearword in an old commit message somewhere, that would IMO not be sufficient to justify force-rewriting the history of that feedstock.